beets/metadata_plugins.py

```diff
@@ -48,30 +48,18 @@ def item_candidates(*args, **kwargs) -> Iterable[TrackInfo]:
         yield from plugin.item_candidates(*args, **kwargs)
 
 
-def album_for_id(_id: str) -> AlbumInfo | None:
-    """Get AlbumInfo object for the given ID string.
-
-    A single ID can yield just a single album, so we return the first match.
-    """
+@notify_info_yielded("albuminfo_received")
+def albums_for_ids(_id: str) -> Iterable[AlbumInfo]:
```

Comment:
Should be named `albums_for_id` if we keep the current call signature. Btw. this seems like a breaking change to me. Can we do this without a major version increment?